README.md

```diff
@@ -147,7 +147,9 @@ If you want to run OpenEdge, you should install docker and python application.
    ```
 
 2. Install python and python runtime requirements
-   
+
+> If you want to run OpenEdge with **Native Process** Mode you should install python 2.7 on your machine with the modules it depends on.
```

Comment:
If you want to run OpenEdge in **native process** mode you should install python 2.7 and some packages depended by python27 runtime.